admin/services/data-access/access-arrangements.data.service.js

```diff
@@ -17,4 +20,39 @@ accessArrangementsDataService.sqlFindAccessArrangements = async function () {
   return sqlService.query(sql)
 }
 
-module.exports = accessArrangementsDataService
+/**
+ * Find access arrangement by pupil Id
+ * @returns {Promise<Array>}
+ */
+accessArrangementsDataService.sqlFindAccessArrangementByPupilId = async function () {
+  const sql = `
+  SELECT 
+    id, 
+    code, 
+    description
+  FROM ${sqlService.adminSchema}.[accessArrangements]
+  WHERE`
```

Comment:
unfinished?

Reply:
@jon-shipley it's removed since it was never used.